### ISSUE TITLE
Can't get chunkwm to work.

### ISSUE THREAD

**Issue by splintah** [koekeishiya/chunkwm#218]
Since I updated to version 2.3.0, chunkwm (installed with `homebrew`) stopped working for me. I updated the plugin directory, because of the changes made in the formula, to `/usr/local/opt/chunkwm/share/chunkwm/plugins`. I checked whether the plugins were in that dir, and they were.
I tried reinstalling, using the example `chunkwmrc` file, but nothing worked.
The error log (`/usr/local/var/log/chunkwm/chunkwm.err.log`) only shows `chunkwm: failed to initialize daemon! abort..`.
You can find my config here: [chunkwmrc](https://github.com/koekeishiya/chunkwm/files/1275287/chunkwmrc.txt) (uploaded as a `.txt` file because GitHub does not support files without extension).

I hope someone can help me.
Thanks in advance.

**Comment by koekeishiya** [comment]
Some program is blocking the port required by chunkwm (maybe an older instance that was not properly stopped before upgrade ??). Rebooting your system should fix that.

**Comment by splintah** [comment]
Thank you; it worked!
When I updated the formula via homebrew, I did not know the plugin dir was changed, and I loaded the old plugin dir in my config. I tried reinstalling it, and compiling it myself. The issue was that I did not delete the `com.koekeishiya.chunkwm.plist` file in `~/Library/LaunchAgents/`, so it would try starting the self-compiled version.

Anyway, thank you for this (already) amazing program!
